src/NLog/Common/DefaultFileCompressor.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace NLog.Common
+{
+#if NET4_5
+using System.IO;
+using System.IO.Compression;
+#endif
+
+    /// <summary>
+    /// Uses .Net4.5 ZipArchive
+    /// </summary>
+    public class DefaultFileCompressor : IFileCompressor
```

Comment:
- I think is better to only include this class in .Net 45. The bool return is then not needed.
- Please use the `<see` tag for `ZipArchive`